amoro-ams/src/main/java/org/apache/amoro/server/DefaultOptimizingService.java

```diff
@@ -241,10 +241,11 @@ public void ackTask(String authToken, int threadId, OptimizingTaskId taskId) {
   @Override
   public void completeTask(String authToken, OptimizingTaskResult taskResult) {
     LOG.info(
-        "Optimizer {} (threadId {}) complete task {}",
+        "Optimizer {} (threadId {}) complete task {} (status: {})",
         authToken,
         taskResult.getThreadId(),
-        taskResult.getTaskId());
+        taskResult.getTaskId(),
+        taskResult.getErrorMessage() == null ? "SUCCESS" : "FAIL");
```

Comment:
nit: we may reuse `TaskRuntime.Status.SUCCESS` and `TaskRuntime.Status.FAILED` instead of hard coding.